webcomponents/tool/src/main/frontend/packages/sakai-submission-messager/src/SakaiSubmissionMessager.js

```diff
@@ -150,21 +217,20 @@ export class SakaiSubmissionMessager extends SakaiElement {
   }
 
   listRecipients() {
-
+    this.recipientsRequested = true;
     const formData = this.getFormData();
 
     fetch("/direct/gbng/listMessageRecipients.json", { method: "POST", cache: "no-cache", credentials: "same-origin", body: formData })
       .then(r => r.json())
       .then(data => {
-
         this.recipientsToCheck = data;
       });
   }
 
   sendMessage() {
 
     if (!this.subject || !this.body) {
-      this.validationError = "You need to supply a subject and body!";
+      this.validationError = this._i18n.validation_error || "You need to supply a subject and body!";
```

Comment:
if the sendMessage method can only be called after a render then shouldupdate should be able to wait on the translation keys being loaded